velox/connectors/hive/storage_adapters/abfs/tests/AbfsFileSystemTest.cpp

```diff
@@ -167,6 +168,29 @@ void readData(ReadFile* readFile) {
       "ccccc");
 }
 
+TEST_F(AbfsFileSystemTest, openFileForReadWithOptions) {
+  auto hiveConfig = AbfsFileSystemTest::hiveConfig(
+      {{"fs.azure.account.key.test.dfs.core.windows.net",
+        azuriteServer->connectionStr()}});
+  auto abfs = std::make_shared<filesystems::abfs::AbfsFileSystem>(hiveConfig);
+  FileOptions options;
+  options.fileSize = 15 + kOneMB;
+  auto readFile = abfs->openFileForRead(fullFilePath, options);
```

Comment:
How is this test verifying the changes? Would it fail without the changes or if we remove code that processes options.fileSize?

Reply:
The test will not fail if we remove the code that processes `options.sizeSize`. However, since fileSize is not 0, the `ASSERT_EQ(readFile->size(), 15 + kOneMB)` in `readData` should test that the expected file size is being set.